### PR TITLE
Mph baseline

### PR DESCRIPTION
**Pull request by mphprogrammer**
Initial commit for your review.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SECRET_KEY_BASE=04d7b1b9a977348c72838c1003690a096902af98f10de5cfb15835faa7f054fa2b5304e2cb8b024e0623143b34def09d6143efaea5718158d173ccd277b8e148
```
Comment (awong-dev) [inline .env]: We don't want this checked in do we?

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+group :development, :test do
+	gem 'puma', '~> 2.15', '>= 2.15.3'
+end
+
+group :production do
+  gem 'rails_serve_static_assets'
+  gem 'rails_12factor'
+end
+
+gem "brakeman", :require => false
```
Comment (awong-dev) [inline Gemfile]: This should be in a development group I think...maybe development/test?

**Comment by mphprogrammer** [comment]
Doh - I thought I put the .env file in .gitignore! Easy fix.

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+## bundle exec rake doc:rails generates the API under doc/api.
+###############################################################################
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'jquery-rails'
+gem 'jquery-ui-rails'
+gem 'jbuilder', '~> 2.0'
+gem 'sdoc', '~> 0.4.0', group: :doc
+
+# Use CoffeeScript for .coffee assets and views
```
Comment (awong-dev) [inline Gemfile]: Remove all the commented out bits?

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+# gem 'coffee-rails', '~> 4.1.0'
+
+# Turbolinks makes following links in your web application faster. 
+#   Read more: https://github.com/rails/turbolinks
+# gem 'turbolinks'
+
+# See https://github.com/rails/execjs#readme for more supported runtimes
+# gem 'therubyracer', platforms: :ruby
+
+# Use ActiveModel has_secure_password
```
Comment (awong-dev) [inline Gemfile]: Is bcrypt a good idea to have by default?

File: README.rdoc [hunk 5]
```diff
@@ -0,0 +1,28 @@
+== README
```
Comment (awong-dev) [inline README.rdoc]: Our projects seem to do markdown more than rdoc for the main readme. Should we do that here? Is there a benefit to rdoc?  I'm not familiar enough with Ruby idioms to know the tradeoffs.

File: app/assets/javascripts/application.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// compiled file.
+//
+// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
+// about supported directives.
+//
+//= require jquery
+//= require jquery_ujs
+//= require turbolinks
```
Comment (awong-dev) [inline app/assets/javascripts/application.js]: kill turbolinks.

Comment (mphprogrammer) [inline app/assets/javascripts/application.js]: Missed that one, its out of the gem not the application js.

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+ 	<%= split_stylesheet_link_tag "application", :media => "all" %>
+
+	<!--[if lte IE 9]>
+    <link href="/stylesheets/application_split2.css" media="all" rel="stylesheet" type="text/css" />
+	<![endif]-->
+</head>
+<body>
+
+	<!-- Render va header -->
+	<%= render 'va_common/header' %>
```
Comment (awong-dev) [inline app/views/layouts/application.html.erb]: If you look at va_common, main template that does this yield for you. Is that not as good of an idea?

File: bin/setup [hunk 8]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```
Comment (awong-dev) [inline bin/setup]: is this auto-generated or our own code?

Comment (mphprogrammer) [inline bin/setup]: Auto-generated

**Comment by awong-dev** [comment]
Nice!  I added some comments to the files.

Also, I noticed this has 3 commits with the same commit message.  Can you squish them and then give the PR itself a correct title? Since this all goes into change history, keeping the commits messages clean is a nice thing to do.

If you are unfamiliar with the `git commands to do all that, let me know.

**Comment by awong-dev** [comment]
errr...I didn't actually stamp this as okay for merge from a code review standpoint yet. :)

@plusjeff: I think we might want to talk through review/check in procedures and language. For example, assign the PR to your reviewer and the reviewer should explicitly signal approval (Google used "LGTM"...mozilla did r+) and then the approver should do the merge.

**Comment by mphprogrammer** [comment]
Sorry - won't happen again, my misunderstanding. Let me know if there are any changes for the initial commit.

**Comment by awong-dev** [comment]
No need to apologize. I think we crossed wires cause we've never agreed on a standardized procedure. I did give you a review, and I wasn't clear about me not being done.

I'll look at it again tomorrow but I think it's probably all good. In the very very worst case, we can rewrite history on the first commit.

**Comment by mphprogrammer** [comment]
I'll work on a dummy resource locally and see if the initial commit holds together.

**Comment by awong-dev** [comment]
This merge was really weird. It wiped out Jeff's original commit too.  I'm going to rewrite history so that we see both commits. @mphprogrammer this will break your next fetch/merge...

**Comment by mphprogrammer** [comment]
No problem, I can just delete the app locally and clone again.

**Comment by awong-dev** [comment]
cool. update is pushed. Also, it's super late over there! stop working and go to sleep!

**Comment by mphprogrammer** [comment]
Sleep - what is this sleep you speak of?
Thanks Albert - have a great evening!

**Pull request merged**